docs/source/extend_kedro/hooks.md

```diff
@@ -411,7 +406,7 @@ class DataValidationHooks:
                 expectation_suite,
             )
             expectation_context.run_validation_operator(
-                "action_list_operator", assets_to_validate=[batch], run_id=run_id
+                "action_list_operator", assets_to_validate=[batch]
```

Comment:
I was wondering whether GE would still work without the `run_id` argument here. It appears to be an optional argument but I think it might be quite useful for users to keep track of validations done over multiple runs.